source/_integrations/free_mobile.markdown

```diff
@@ -11,21 +11,25 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `free_mobile` platform is using the French mobile operator [Free Mobile](http://mobile.free.fr/) to send SMS to your own cell phone. 
+The `free_mobile` platform uses the French mobile operator [Free Mobile](http://mobile.free.fr/) to send SMS to your own cell phone.
+
+## Setup
```

Comment:
```suggestion
## Prerequisites
```

Reply:
The **Setup** heading is used in documentation of many (>100) integrations, like: [accuweather](https://www.home-assistant.io/integrations/accuweather) or [amazon polly](https://www.home-assistant.io/integrations/amazon_polly).

On the other hand, **Prerequisites** is also used for about 60 integrations.

I'll commit the changes.
Thanks!

Reply:
Yeah, we started using `Prerequisites` some time ago as it better describes the intent.
Setup and Configuration are generally a bit confusing, as they can mean the same (as a matter of fact, we have integrations that describe our configuration procedure with a "Setup" heading.

Reply:
Very clear, and I agree. 
Thanks!